DESIGN.md

```diff
@@ -54,39 +56,113 @@ Usage:
 -e --execute ["text"|script-file|STDIN]
 ```
 
-## Meta Commands
+## Command interface
+
+Users can type three types of entries into the shell:
+
+1. [PartiQL statements](https://partiql.org/)., 
+1. Database commands, and
```

Comment:
I know the `1.` style means the rendering autonumbers for you, but I often read raw md and appreciate the numbers being correct.

Reply:
Fixed